README.rst

```diff
@@ -14,9 +16,47 @@ The following GUI backends are supported:
 - PyQt
 - PySide
 
+**Warning:** In TraitsUI version 5.0 the default GUI backend will change from
```

Comment:
one can use rst admonitions http://docutils.sourceforge.net/docs/ref/rst/directives.html#admonitions), they should be rendered properly by sphinx